ios/XMTPModule.swift

```diff
@@ -70,10 +70,52 @@ extension Conversation {
 }
 
 public class XMTPModule: Module {
-    var clients: [String: XMTP.Client] = [:]
     var signer: ReactNativeSigner?
-    var conversations: [String: Conversation] = [:]
-    var subscriptions: [String: Task<Void, Never>] = [:]
+    let clientsManager = ClientsManager()
+    let conversationsManager = ConversationsManager()
+    let subscriptionsManager = SubscriptionsManager()
+    
+    actor ClientsManager {
+        private var clients: [String: XMTP.Client] = [:]
+
+        // A method to update the conversations
+        func updateClient(key: String, client: XMTP.Client?) {
+            clients[key] = client
+        }
+
+        // A method to retrieve a conversation
+        func getClient(key: String) -> XMTP.Client? {
+            return clients[key]
+        }
+    }
```

Comment:
Instead of wrapping these objects, could we just keep the dictionaries and use the `updateValue` function:

```swift
await clients.updateValue(client, forKey: "some address")
``` 

then we can just use the normal subscript getters.